R/create_reference_values.R

```diff
@@ -14,81 +15,110 @@
 #'
 #' @export
 
-create_reference_values <- function() {
+##FIXME: remove this function if check_values_capture is updated.
```

Comment:
Can now be removed?

Reply:
The first part of that function, so the lines concerning brood data, can be removed, but the lines concerning capture data have to stay because the check_values_capture function has not been updated yet.